docker/Dockerfile

```diff
@@ -14,18 +14,22 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 #
-
-FROM centos:7.6.1810
+ARG java_image_tag=8-jdk-focal
```

Comment:
not sure use lowercase is a good practice or not for `ARG`

Reply:
We refer styles from the Spark Dockerfile